lib/slipstream/configuration.ex

```diff
@@ -27,9 +27,16 @@ defmodule Slipstream.Configuration do
       type: {:list, {:custom, __MODULE__, :parse_pair_of_strings, []}},
       default: []
     ],
+    serializer: [
+      doc: """
+      A serializer module which exports at least `encode!/1` and `decode!/2`.
+      """,
+      type: :atom,
+      default: Slipstream.Serializer.PhoenixSocketV2Serializer
+    ],
     json_parser: [
       doc: """
-      A JSON parser module which exports at least `encode/1` and `decode/2`.
+      A JSON parser module which exports at least `encode!/1` and `decode!/1`.
```

Comment:
This makes a breaking change for users who use custom JSON parser which doesn't have `decode!/1`.

### before this PR

The original line is not fit the actual codes. The codes expected `encode!/1` and `decode/1`. This was just document error. Custom JSON parser users know that and implement them to fit the codes.

### after this PR

This PR codes expect `decode!/1` for json_parser. So this change should be written in CHANGELOG.